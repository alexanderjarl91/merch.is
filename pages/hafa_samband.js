import React from "react";
import styles from "../styles/Hafa_samabnd.module.css";
import Layout from "../components/Layout";

function hafa_samband(props) {
  return (
    <>
      <Layout>
        <div className={styles.container}>
          <p className={styles.hafa_samaband_h1}>
            Hafðu samband, við elskum að fá ný skilaboð
          </p>
          <form className={styles.contact_form} action="/" method="POST">
            <label className={styles.contact_label} for="name">
              Nafn
            </label>
            <input
              className={styles.contact_input}
              type="text"
              required
            ></input>

            <label className={styles.contact_label} for="email">
              Netfang
            </label>
            <input
              className={styles.contact_input}
              type="text"
              required
            ></input>

            <label className={styles.contact_label}>Skilaboð</label>
            <textarea
              rows="4"
              className={styles.contact_textarea}
              required
            ></textarea>

            <button className={styles.contact_btn} type="submit">
              SENDA
            </button>
          </form>
        </div>
      </Layout>
    </>
  );
}

export default hafa_samband;
