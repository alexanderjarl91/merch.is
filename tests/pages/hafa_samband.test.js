import { shallow } from "enzyme";
import hafa_samband from "../../pages/hafa_samband";

it("expect hafa_samband component to match Snapshot", () => {
  expect(shallow(<hafa_samband />)).toMatchSnapshot();
});

it("expect to render component hafa_samband", () => {
  expect(shallow(<hafa_samband />).length).toEqual(1);
});
