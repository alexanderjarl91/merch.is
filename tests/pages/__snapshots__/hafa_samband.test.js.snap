// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect hafa_samband component to match Snapshot 1`] = `ShallowWrapper {}`;
